test/test_integration.py

```diff
@@ -24,7 +24,7 @@ def test_start_hint():
     res = requests.get("http://localhost:8080")
 
     assert res.status_code == 200
-    assert "Login" in res.content.decode("UTF-8")
+    assert "Naomi" in res.content.decode("UTF-8")
```

Comment:
Unrelated to the current ticket but we no longer land on a login page and this build hasn't been run since that change was made.